src/Units/MoveTypes/Flight.java

```diff
@@ -6,8 +6,8 @@
   public Flight()
   {
     // format is [weather][terrain]
-    int[][] tempCosts = { { 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1 }, { 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1 },
-        { 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2 }, { 1, 1, 1, 3, 1, 1, 1, 1, 1, 1, 1 } };
+    int[][] tempCosts = { { 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1 }, { 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1 },
```

Comment:
Ugh. I don't like having to do manual data entry in opaque 2D arrays to support new terrain/movement types. Add this to the list of things to revisit.